src/components/IdentityManager/IdentityManager.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Subject } from 'rxjs'
+
+const identityEventTypes = {
+  CLEAR: 'CLEAR',
+  IMPORT: 'IMPORT',
+  MODIFY: 'MODIFY',
+}
+
+// An events subject
+// { type: '<string>', address: <string> }
+const identityEvents$ = new Subject()
```

Comment:
Using RxJS in aragon/aragon is something we’ve been avoiding until now (by trying to keep it all in `aragonjs-wrapper.js`)… is the intention to start using it more in our React apps? If so, I think we should discuss it so we can align on this.

Reply:
Originally we used an event emitter, Daniel suggested using an observable since we are already using `rxjs` and I agree, what would you propose we do instead?

Reply:
It should be tried to validate it, but a solution I can see would be to wrap it all in a module, exposing a provider component + React Hooks.

Reply:
That is what is going on here, the `Observable` is instantiated inside a `Provider`, then `Consumer` have access to it and listen to events when something happens (import, export or modify).

Reply:
My concern is that it exposes `identityEvents$`, which requires to be “plugged” everywhere it’s being used (`.subscribe()`, `.unsusubscribe()`, etc.), instead of a React mechanism.

Reply:
I tried to create a callback/hook mechanism but couldn't make it work, this is a situation where an event emitter (observable in `rxjs`) is solves the problem.

Reply:
So we'd introduce a custom hook which we pass update callbacks to and the hook will handle all the rxjs specific subscriptions and cleanup.

@bpierre Did I understand your suggestion correctly?

Reply:
That’s fine, let’s discuss this later then. 👍 To be clear I am not requesting any change, I just wanted to discuss the usage of RxJS in our apps, which is something we were trying to avoid up to this point.

About this case specifically, it could be nice to add some comments in the code about the problems you encountered with React hooks / other React mechanisms that made you decide to use `rxjs` instead.

Reply:
@2color Exactly! But @AquiGorka tried it already and couldn’t make it work, so we can explore this idea later. 👍